packages/big-design/src/components/Worksheet/Cell/Cell.tsx

```diff
@@ -89,13 +89,14 @@ const InternalCell = <T extends WorksheetItem>({
     switch (type) {
       case 'select':
         return <SelectEditor cell={cell} isEdited={isEdited} onChange={handleChange} options={options} />;
-      // Default to TextEditor
+      case 'checkbox':
+        return <CheckboxEditor cell={cell} onChange={handleChange} />;
       default:
         return isEditing ? (
           <TextEditor cell={cell} isEdited={isEdited} onBlur={handleBlur} onKeyDown={handleKeyDown} />
         ) : (
           // In case of NaN casting to string
-          <Small color="secondary70">{value.toString()}</Small>
+          <Small color="secondary70">{value ? value.toString() : ''}</Small>
```

Comment:
I think NaN will cast to "NaN" - is this `toString()` intended to avoid that?

Reply:
Is it intended to do that.

Reply:
👍 Ah, nvm, I was thinking of a different implementation (`||`) which would have returned `"NaN"`, but a ternary avoids that.